README.md

```diff
@@ -6,37 +10,35 @@
 [![commits last month](https://img.shields.io/github/commit-activity/m/illinoisdata/ElasticNotebook)](htps://github.com/illinoisdata/kishu)
 [![GitHub stars](https://img.shields.io/github/stars/illinoisdata/ElasticNotebook)](htps://github.com/illinoisdata/kishu)
 --->
+</div>
 
-# Kishu: Versioned and Undoable Notebook System
+Kishu is a system for versioning notebook session states in IPython-based notebooks (e.g. JupyterLab, Jupyter Hub). It efficiently and transparently checkpoints both variable and code states of notebook sessions, enabling users to undo cell executions and manage branching states containing objects such as ML models, plots, and dataframes via Git-like commits and checkouts.
 
-<br>
 <div align="center">
-<img width="350" src="docs/images/kishu_with_background.png"/>
+<img width="960" src="docs/images/kishu_frontpage.png"/>
 </div>
 <br>
 
-Kishu is a system for intelligent versioning of notebook session states on Jupyter-based platforms (e.g. JupyterLab, Jupyter Hub). Kishu efficiently creates checkpoints of both the variable and code states of a notebook session, allowing users to both undo cell executions and manage branching states containing objects such as machine learning models, plots, and dataframes through a Git-like commit and checkout interface.
-
 ## Getting Started
 
-Kishu can be installed from [PyPI](https://pypi.org/project/kishu/):
+Install Kishu directly from [PyPI](https://pypi.org/project/kishu/):
 
 ```bash
-pip install kishu jupyterlab_kishu
+pip install jupyterlab_kishu
```

Comment:
Might be confusing since PyPI link is pointing to `kishu` but command is `jupyterlab_kishu`. I'd keep both refs on both places, with the right concise description